src/test/java/uk/gov/pay/api/utils/mocks/ConnectorDDMockClient.java

```diff
@@ -35,63 +35,81 @@ public void respondWithChargeFound(
             String description, String reference, String email, String paymentProvider, String createdDate,
             String chargeTokenId) {
         String chargeResponseBody = buildPaymentRequestResponse(amount, chargeId, state, returnUrl,
-                description, reference, email, paymentProvider, createdDate,
-                validGetLink(chargeLocation(gatewayAccountId, chargeId), "self"),
-                validGetLink(nextUrl(chargeId), "next_url"),
-                validPostLink(nextUrlPost(), "next_url_post", "application/x-www-form-urlencoded",
-                        new HashMap<String, String>() {{
-                            put("chargeTokenId", chargeTokenId);
-                        }}));
+                                                                description, reference, email, paymentProvider, createdDate,
```

Comment:
wtf

Reply:
@DanailMinchev how could you allow this

Reply:
@simad sorry :D evil auto-formatting and changed settings in the last minute